dataset_generation/hi_wikisplit/Scraper.py

```diff
@@ -0,0 +1,174 @@
+import logging
+import requests
+import os
+import sys
+import shutil
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+class Scraper:
+    """
+    Uses the wikimedia api to get the revision histories of the titles 
+    provided in the title dump file.
+    Format varies between wikipedia languages
```

Comment:
what exactly does this mean? If we want to sort of scrape for marathi language, do we need to change the code?

Reply:
mostly not. but when i tried it for hindi and english, slightly different html tags were used to indicate the edited lines in the history.

Reply:
ok. can we please mention in the readme file to specifically take care in this file

Reply:
yeah